components/index.js

```diff
@@ -231,3 +231,6 @@ export UtilityIcon from './utilities/utility-icon';
 
 export SLDSTree from './tree';
 export Tree from './tree';
+
+// Library utilities
+export log from '../utilities/log';
```

Comment:
what is this export used for?

Reply:
Adds a `log` utility that will route logs to the correct environment such as Storybook actions, `console.log` (on the docs site), or a custom function that we could extend in the future. 

My mistake that was missing from the description. This should enable us to use the same stories in Storybook as in the doc website and place the complexity of the multi-environment output within this public function.